src/styles/StyleUtils.js

```diff
@@ -404,6 +404,16 @@ function parseStyleAsArray(styleParam) {
     return _.isArray(styleParam) ? styleParam : [styleParam];
 }
 
+/**
```

Comment:
Can we add a description to this function as to why it exists. It's pretty unclear what the purpose of this is on its own.